tls/s2n_resume.h

```diff
@@ -23,14 +23,18 @@
 #define S2N_TLS13_FIXED_STATE_SIZE            21
 #define S2N_TLS13_FIXED_EARLY_DATA_STATE_SIZE 3
 
-#define S2N_TLS_SESSION_CACHE_TTL      (6 * 60 * 60)
```

Comment:
Our clang-format job insists that these by reformatted. 🤷 

`codebuild/bin/clang_format_changed_files.sh`

Reply:
That's because it's interpreting your new comment as basically whitespace. It wants all the defines in a block aligned. Before they were all aligned to  S2N_TLS12_TICKET_SIZE_IN_BYTES, but now that's considered part of a different block. You could fix it by putting your define first. Or last, but then your define won't align with the others in the block.